.github/workflows/security_semgrep.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     name: Semgrep Analyze
     runs-on: ubuntu-latest
     container:
-      image: returntocorp/semgrep
+      image: returntocorp/semgrep@sha256:e7216f0ec1eaa032f470733e33d4a49d556f04b82fdce5e066599c38c1334c32 # 1.47.0
```

Comment:
shouldn't we also swap over to the `semgrep/semgrep` version?

Reply:
@tdonaworth So I looked at and tried that but it seemed to bomb out because even though their GitHub org has changed, in the docker registry it's still `returntocorp`. A 404 error is returned. I could have been missing something but that's what I experienced anyway